pkg/sql/logictest/testdata/logic_test/show_source

```diff
@@ -221,3 +221,25 @@ CREATE INDEX ix ON foo(x); SELECT * FROM [SHOW TESTING_RANGES FROM INDEX foo@ix]
 ----
 Start Key  End Key  Replicas  Lease Holder
 NULL       NULL     {1}       1
+
+query IIITTTT colnames
+SELECT * FROM [SHOW SESSION TRACE]
+----
+txn_idx  span_idx  message_idx  timestamp  duration  operation  message
+
+# Tracing
+
+statement ok
+BEGIN; SET trace = on; SELECT 1; SET trace = off; COMMIT
```

Comment:
Added a comment to clarify.

We can merge but since the following *two* statements use the trace I thought it was clearer to place the set-up state in a distinct line.